docs/faq.rst

```diff
@@ -391,6 +391,17 @@ How can I configure and debug the exact version exported by my package?
 
     (This requires the ``setuptools-scm`` package is installed in your environment)
 
+Why do I see the error ``AssertionError: cant parse version ...``?
+    There are some limitations with ``setuptools-scm`` and in some edge cases
+    it might fail to parse specific tags.
+    If that is happening to your project (other than removing the problematic tag),
+    you can try to re-run the relevant command after setting the
+    ``SETUPTOOLS_SCM_PRETEND_VERSION`` environment variable to overwrite the
+    default behavior.
```

Comment:
Giving here an example might help. So `export SETUPTOOLS_SCM_PRETEND_VERSION=v1.4` in Bash, right?

Reply:
For some reason, GitHub separated my reply in a different comment: https://github.com/pyscaffold/pyscaffold/pull/651#discussion_r901065806.